diff --git a/node_modules/react-native-branch/android/src/main/java/io/branch/rnbranch/RNBranchModule.java b/node_modules/react-native-branch/android/src/main/java/io/branch/rnbranch/RNBranchModule.java
index 1880a5b..9de6b07 100644
--- a/node_modules/react-native-branch/android/src/main/java/io/branch/rnbranch/RNBranchModule.java
+++ b/node_modules/react-native-branch/android/src/main/java/io/branch/rnbranch/RNBranchModule.java
@@ -542,10 +542,12 @@ public class RNBranchModule extends ReactContextBaseJavaModule {
                 ShareSheetStyle shareSheetStyle = new ShareSheetStyle(mContext, messageHeader, messageBody)
                         .setCopyUrlStyle(mContext.getResources().getDrawable(android.R.drawable.ic_menu_send), "Copy", "Added to clipboard")
                         .setMoreOptionStyle(mContext.getResources().getDrawable(android.R.drawable.ic_menu_search), "Show more")
-                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.EMAIL)
                         .addPreferredSharingOption(SharingHelper.SHARE_WITH.TWITTER)
-                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.MESSAGE)
-                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.FACEBOOK);
+                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.FACEBOOK_MESSENGER)
+                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.GMAIL)
+                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.SNAPCHAT)
+                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.INSTAGRAM)
+                        .addPreferredSharingOption(SharingHelper.SHARE_WITH.WHATS_APP);
 
                 BranchUniversalObject branchUniversalObject = findUniversalObjectOrReject(ident, mPm);
                 if (branchUniversalObject == null) {
